src/main/java/frc/robot/Robot.java

```diff
@@ -319,6 +319,16 @@ private void handleState() {
           .setBoolean(m_isInControlPanelMode);
     }
 
+    // If launching mode is enabled and the robot is driven, disable it.
+    if ((Math.abs(m_controllerDrive.getRawAxis(DriveStation.kIdAxisLeftY)) > 0.5
```

Comment:
This would be more optimized if it was something like:
```java
if (Math.abs(m_controllerDrive.getRawAxis(DriveStation.kIdAxisLeftY)) > 0.5 || Math.abs(m_controllerDrive.getRawAxis(DriveStation.kIdAxisRightX)) > 0.5))
  if (m_isInControlPanelMode)
    // ...
  else if (m_isInLaunchingMode)
    // ...
```

Reply:
Agreed. I will implement these changes.